samples/timelapse-sample/CMakeLists.txt

```diff
@@ -0,0 +1,53 @@
+set(TARGET timelapse-sample)
+SET(CORE_FILES
+	timelapse_summary.cpp
+)
+SET(CORE_FILES_H
+	timelapse_summary.h
+)
+SET(SOURCE
+  ${CORE_FILES}
+  ${CORE_FILES_H}
+)
+set(UT_FILE
+	timelapse_summary_test.cpp
+)
+find_library(APRA_PIPES_LIB NAMES aprapipes)
+add_library(example_aprapipes STATIC ${SOURCE})
+add_executable(runner_test ${UT_FILE})
```

Comment:
I dont like this. Why build a executable for this test. It already is added in aprapipesut executable. We need a separate test. Maybe the main need to call a method which can be called from tests also

Reply:
As discussed I have added an executable with the source files and the test file for the sample.